charts/operator-wandb/charts/weave/templates/deployment.yaml

```
@@ -89,13 +87,32 @@ spec:
           volumeMounts:
             - name: cache
               mountPath: /vol/weave/cache
+
+        - name: {{ include "weave.fullname" . }}-cache-clear
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          command: ["python", "-m", "weave.clear_cache"]
+            
+          env:
+            - name: WEAVE_LOCAL_ARTIFACT_DIR
+              value: /vol/weave/cache
+            - name: WEAVE_CACHE_CLEAR_INTERVAL
+              value: "{{ .Values.clearCache.hourInterval | default 24 }}"
+
+            {{- include "weave.extraEnv" (dict "global" .Values.global "local" .Values) | nindent 12 }}
+            {{- include "wandb.extraEnvFrom" (dict "root" $ "local" .) | nindent 12 }}
+
+          volumeMounts:
+            - name: cache
+              mountPath: /vol/weave/cache
+
       volumes:
         - name: cache
-          persistentVolumeClaim:
-            claimName: {{ template "weave.fullname" . }}
+          emptyDir: 
+            sizeLimit: {{ .Values.cache.size | default "20Gi" }}
+            medium: {{ .Values.cache.medium | default "" }}
```

```suggestion
            sizeLimit: {{ .Values.cache.size }}
            medium: {{ .Values.cache.medium }}
```

Defaults should be set in the values,yaml